### DISCUSSION TITLE
Ajout de quêtes

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#603] [Idées acceptées / déjà implémentées]
Différentes missions pour le joueur, donnant lieux à divers récompenses
![](https://cdn.productboard.com/production/attachments/e0abb5e41cf20ff62ec4f8ee12c433380a408f4af2de1cb0ea2cee63453ab7bc/portal_cover/Capture_d%E2%80%99%C3%A9cran_de_2021-03-03_15-02-49.png)

**Comment by Feiryn** [comment]
On peut fermer ça non ?

**Reply by BastLast** [reply]
done
